networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/topics/validation/AttestationValidator.java

```diff
@@ -111,19 +113,19 @@ ValidationResult singleOrAggregateAttestationChecks(final Attestation attestatio
     final UnsignedLong currentTimeMillis = secondsToMillis(recentChainData.getStore().getTime());
     if (isCurrentTimeAfterAttestationPropagationSlotRange(currentTimeMillis, attestation)
         || isFromFarFuture(attestation, currentTimeMillis)) {
-      return INVALID;
+      return IGNORE;
     }
     if (isCurrentTimeBeforeMinimumAttestationBroadcastTime(attestation, currentTimeMillis)) {
-      return SAVED_FOR_FUTURE;
+      return SAVE_FOR_FUTURE;
     }
 
     // The block being voted for (attestation.data.beacon_block_root) passes validation.
     // It must pass validation to be in the store.
-    // If it's not in the store, it may not have been processed yet so save for future.
+    // If it's not in the store, it must not have passed validation.
```

Comment:
Probably should restore the old version of this comment:
```suggestion
    // If it's not in the store, it may not have been processed yet so save for future.
```

Reply:
Good catch. Done.